### PR TITLE
Order creation

### PR DESCRIPTION
**Pull request by mkasztelnik**
Basic ordering capabilities. Logged in user can click `Order` button on service page and as a conclusion ordering process begins. At the beginning `Order` record is created in our database with first `OrderChange` record. This change order status into `created` and set appropriate message. At the end of order creation process, the delayed job is triggered to register order in an external system (JIRA in our case). 

`Order::RegisterJob` invokes `Order::Register` service which is a placeholder for JIRA integration. At the end, it adds new order change (change status from `created` into `registered` with the dedicated message). 

This PR is related to following user stores:
  * #44 
  * #42 
  * #41
  * #40

I'm not selecting these issues as "closable" because my implementation is not styled and views are ugly - thus according to EU "read" understanding this is not ready.

### CODE REVIEW

File: app/views/orders/_order.html.haml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+%dt
+  Service:
+  = link_to order.service.title, order.service
```
Comment (bwilk) [inline app/views/orders/_order.html.haml]: here you should link to order not service

File: app/models/order.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Order < ApplicationRecord
+  STATUSES = {
+    created: "created",
+    registered: "registered",
```
Comment (bwilk) [inline app/models/order.rb]: what does 'register' mean here?

File: app/services/order/register.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  def call
+    register_in_jira! &&
+    update_status!
+  end
+
+  private
+
+    def register_in_jira!
+      # TODO: Jira integration. This method should throw error on errors which
```
Comment (bwilk) [inline app/services/order/register.rb]: @michal-szostak take this into account while solving #29

File: app/models/order.rb (hunk 2 above)
Comment (mkasztelnik) [reply app/models/order.rb]: An order is registered in 3rd party service (e.g. JIRA)

File: app/views/orders/_order.html.haml (hunk 1 above)
Comment (mkasztelnik) [reply app/views/orders/_order.html.haml]: done

**Comment by mkasztelnik** [comment]
All reported issues solve, this is ready to be merged after @bwilk approve.

**Pull request merged**